css/_videolayout_default.scss

```diff
@@ -269,6 +270,16 @@
     z-index: $zindex2;
 }
 
+#alwaysOnTop .displayname {
+    font-size: 15px;
+    position: inherit;
```

Comment:
What position does it inherit?

Reply:
Wanted to override the position: absolute. Is it better to use position: initial? Or should I use something else?

Reply:
So what is the position now? Do you need left: 0 and right: 0 still?

Reply:
It inherits position: static according to the inspector and I need top:0 and left:0 to override the previous values which was some percents if I recall correctly.

Reply:
I'm not sure then that top and left would affect a static element. The overrides may not be necessary.